tests/test_quick_run_config_generator.py

```diff
@@ -910,11 +910,12 @@ def _get_next_run_config_ensemble(self,
         Also,
         - sequence batching should be on for model 1
         - dynamic batching should be on for model 2
+        - cpu_only should be set for model 2
```

Comment:
This comment (model 2) conflicts with the variable name on line 943 (model_config1)

Reply:
I've change the names to be model A/B to clear this up.